WordPress/Classes/ViewRelated/Post/Prepublishing Nudge/PrepublishingHeaderView.swift

```diff
@@ -53,8 +53,12 @@ class PrepublishingHeaderView: UIView, NibLoadable {
     }
 
     private func configureBackButton() {
-        backButtonView.isHidden = true
-        backButton.setImage(.gridicon(.chevronLeft, size: Constants.backButtonSize), for: .normal)
+        closeButtonView.isHidden = true
+        closeButton.setImage(.gridicon(.cross, size: Constants.backButtonSize), for: .normal)
+        closeButton.accessibilityHint = Constants.doubleTapToDismiss
```

Comment:
When I highlight the button using voice over it just says "Button", should we also set the `accessibilityLabel` to "Close" so when highlighted it'll read as "Close, button"

Reply:
Done!